cmd/daemon/kubernetes/pods.go

```diff
@@ -128,7 +128,7 @@ func (p *PodInformer) handle(e interface{}) {
 	}
 
 	if isPodOOMKilled(pod) {
-		log.Infof("Pod: %s was OOMKilled owned by squad %s", pod.Name, getCodeOwnerSquad(pod.Labels))
+		log.With("killedPodNamespace", pod.Namespace).Infof("Pod: %s was OOMKilled owned by squad %s", pod.Name, getCodeOwnerSquad(pod.Labels))
```

Comment:
this seems like a strange name for a label?

Or we can get a little more generic name for the pod or put it in the template itself.

We could just as well serialize all the information about the pod and put that in the log as well, then we can extract the parts we need.

Reply:
@kjuulh I am open to suggestions for a different field name, as long as it doesn't collide with existing labels. I don't think it makes sense to log everything (you have the final say as you own the service)

Reply:
Maybe we could call it `targetPodNamespace`, we could also move pod down so that it would be called `targetPodName`.

Reply:
@kjuulh What do you think about b8df41c?

Reply:
Looks good